src/assets/index.js

```diff
@@ -0,0 +1,35 @@
+import AOE from './AOE.jpg';
+import blackflag from './blackflag.jpg';
+import RVG from './RVG.jpg';
+import GameSpot from './GameSpot.jpg';
+import SergiuHellDragoonHQ from './SergiuHellDragoonHQ.jpg';
+import DKGames from './DKGames.jpg';
+import igcompany from './igcompany.jpg';
+import PlayStation from './PlayStation.jpg';
+import Throneful from './Throneful.jpg';
+import Shirrako from './Shirrako.jpg';
+import Fennec from './Fennec.jpg';
+import DIZEUUL from './DIZEUUL.jpg';
+import MrEdxwx from './MrEdxwx.jpg';
+import xbox from './xbox.jpg';
+import ign from './ign.jpg';
+import rockstar from './rockstar.jpg';
```

Comment:
Try to have same naming convention of all of your images.

Reply:
I have used the video channel names to avoid confusion in the database while using these values.